src/dbus/message.c

```diff
@@ -11,6 +11,7 @@
 #include "dbus/message.h"
 #include "dbus/protocol.h"
 #include "util/fdlist.h"
+#include "util/log.h"
 #include "util/error.h"
```

Comment:
My eyes! They hurt! This is not sorted alphanumerically... (I will push a fix myself)